validation/deleate_only_create_resources/delte_only_create_resources.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	tap "github.com/mndrix/tap-go"
+	"github.com/mrunalp/fileutils"
+	rspec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/opencontainers/runtime-tools/specerror"
+	"github.com/opencontainers/runtime-tools/validation/util"
+	uuid "github.com/satori/go.uuid"
+)
+
+func main() {
+	t := tap.New()
+	t.Header(0)
+
+	// Create a cgroup
+	cgPath := "/sys/fs/cgroup"
+	pidsPath := filepath.Join(cgPath, "pids")
```

Comment:
This will not work with cgroup-v2 because the paths are different but that's a larger in OCI.

Reply:
I don't know much about cgroup-v2, so should we do validation here in two different cases?

I will learn about cgroup-v2 later. There is still much work to be done about cgroup-v2 in the runtime-tool.

Reply:
A lot about the spec is cgroup-v1 only. I don't think we can do add support for v1 and v2 everywhere at once. So I don't think this should block this PR.